cmd/ctr-remote/sampler/sampler.go

```diff
@@ -361,21 +361,31 @@ func runContainer(ctx context.Context, bundle string) error {
 	}()
 
 	// Wait until context is canceled or signal is detected
-	log.G(ctx).Debugf("waiting for the termination of container %q", id)
 	sc := make(chan os.Signal, 1)
 	signal.Notify(sc,
 		syscall.SIGHUP,
 		syscall.SIGINT,
 		syscall.SIGTERM,
 		syscall.SIGQUIT)
 	defer signal.Stop(sc)
-	select {
-	case <-done:
-		return nil
-	case <-ctx.Done():
-		log.G(ctx).Info("context canceled")
-	case <-sc:
-		log.G(ctx).Info("signal detected")
+	if ignoreCtxCancel {
+		log.G(ctx).Debugf("press Ctrl+C to terminate the container %q", id)
```

Comment:
Can we make it `Infof`? This information seems important for user.

Reply:
Sure!